content/before-you-open-a-support-ticket.md

```diff
@@ -20,91 +20,95 @@ section: getting-help
 tableOfContents: true
 ---
 
-This article is designed to provided a range of self-help resources from our other support articles, all in one place. Please review this article and familiarize yourself with the kind of information we will need to help diagnose your issue. Instructions for some of the testing we perform on support tickets is included.
-
-Any information, or test results, you can provide when opening your support case, will help us resolve your issue faster.
-If you’ve already done some of these items, just let us know! Otherwise, feel free to skip that item in this list, and check the others. Please do provide the outcomes of any testing you've already done.
-
-**NOTE:** You can use this article as a guide for your messaging when opening a support case. Please answer as many of the questions as you can.
-
-The first message sent on a ticket does not support file attachments; however, the second message will.
-
-If you're having difficulty attaching a file, you can also email the file to info@system76.com. Please be sure to tell us on the ticket if you have sent an email attachment.
-
-If you encounter problems signing into your account, or opening a ticket, please [follow these steps](/articles/clear-cache-and-cookies)
+This article combines several self-help articles resolving common issues experienced by the Support team. This also includes standard troubleshooting procedures we perform for all support requests. Please review this information. Any background and testing information you can provide up front helps us more efficiently diagnose and resolve software and hardware related issues.
 
 ## Before We Begin
 
 ### 1. Do you have a current backup of your files?
 
-Depending on how time sensitive your issue is, the fastest resolution may be to reinstall the operating system. Re-imaging is much easier to do if we know that your important files are safe.
+If your issue is time-sensitive, reinstalling the operating system may provide the fastest resolution. Be sure to backup all important data before performing any tests or contacting Support. Re-imaging is much easier to do if we know that your important files are safe.
```

Comment:
Maybe it would be good to mention the Refresh OS option here? I'm thinking something like, "If your issue is time-sensitive, reinstalling or refreshing the operating system..."